packages/dev/gui/src/2D/controls/rectangle.ts

```diff
@@ -29,6 +31,18 @@ export class Rectangle extends Container {
     public get cornerRadius(): number {
         return this._cornerRadius;

```

Comment:
i can this can be removed and we can use the first element of the array? especially if it is a private member

Reply:
Oh, you're right, if you just keep the getter and setter returning/receiving the same type as before, we don't break back-compat. Sorry for having missed this before @J3F31 !

Reply:
So if I understand correctly... Only one variable should be used for the corner radius (vector4 or array), and the get and set functions that already exist should take the first value of the corner radius var?